lib/validators/stripped_length_validator.rb

```diff
@@ -26,12 +26,14 @@ def validate_each(record, attribute, value)
     self.class.validate(record, attribute, value, range)
   end
 
-  def self.get_sanitized_value(value)
+  def self.get_sanitized_value(value, strip_uploads: false)
     value = value.dup
     value.gsub!(/<!--(.*?)-->/, "") # strip HTML comments
     value.gsub!(/:\w+(:\w+)?:/, "X") # replace emojis with a single character
     value.gsub!(/\.{2,}/, "…") # replace multiple ... with …
     value.gsub!(/\,{2,}/, ",") # replace multiple ,,, with ,
+    value.gsub!(/!\[.*\]\(.+\)/, "") if strip_uploads
```

Comment:
Pathological strings won't cause a DoS here, because we use a global regex timeout.